cilium-cli/connectivity/tests/to-cidr.go

```diff
@@ -51,3 +52,16 @@ func (s *podToCIDR) Run(ctx context.Context, t *check.Test) {
 		}
 	}
 }
+
+func ipToName(ip string) string {
+	ipWithoutSep := strings.ReplaceAll(ip, ".", "")          // IPv4 separator
+	ipWithoutSep = strings.ReplaceAll(ipWithoutSep, ":", "") // IPv6 separator
+	return fmt.Sprintf("external-%s", ipWithoutSep)
+}
+
+func ipToURL(ipString string) string {
+	if ip := net.ParseIP(ipString); ip != nil && ip.To4() == nil {
+		ipString = fmt.Sprintf("[%s]", ipString) // Avoid parsing IPv6 last ":" as port
+	}
```

Comment:
Same here, let's use `net/netip`:

```suggestion
	if ip, err := netip.ParseAddr(ipString); err == nil && ip.Is6() {
		ipString = fmt.Sprintf("[%s]", ipString) // Avoid parsing IPv6 last ":" as port
	}
```